src/pyramid_googleauth/templates/login_failure.html.jinja2

```diff
@@ -0,0 +1,3 @@
+<h1>You are not authorized to view these pages</h1>
```

Comment:
This is currently not configurable by the clients 

Not sure if we should leave it as is, or maybe allow to configure the teamplate? I probably would be easy to overwrite the whole view.